### PR TITLE
Settings via context menu

### PR DESCRIPTION
**Pull request by staff0rd**
A user may type `settings` to access the settings ui, however it is likely they will be using the mouse on that ui, so an option to use the mouse earlier rather than the keyboard to open the ui should exist.

This change extends the existing `MainWindow.xaml` ContextMenu to include `Settings` menu item, which upon clicking will open the `SettingWindow`.

### CODE REVIEW

**Comment by bao-qian** [comment]
why you can't just click `settings` item using mouse?

**Comment by staff0rd** [comment]
Do you mean via the task tray?  It is far quicker to open wox with the hotkey, and right click settings than it is to search for it in the task tray, or type settings.  This amplified when switching in and out of settings after a new install while adding hotkeys.  Additionally, other launchers generally have this feature - I'm coming from executor for example.  If you don't believe your users need this speed, close the PR and I'll keep it in my fork.

File: Wox/MainWindow.xaml [hunk 1]
```diff
@@ -29,7 +29,17 @@
...
                      PreviewDragOver="TbQuery_OnPreviewDragOver" AllowDrop="True"
-                     x:Name="tbQuery" />
+                     x:Name="tbQuery">
+                <TextBox.ContextMenu>
+                    <ContextMenu>
+                        <MenuItem Command="ApplicationCommands.Cut"/>
+                        <MenuItem Command="ApplicationCommands.Copy"/>
+                        <MenuItem Command="ApplicationCommands.Paste"/>
+                        <Separator />
+                        <MenuItem Header="Settings" Click="menuSettings_Click" />
```
Comment (bao-qian) [inline Wox/MainWindow.xaml]: I think it would be better to call `menuSettings_Click` as `OnContextMenusForSettingsClick`.

**Comment by bao-qian** [comment]
Ok, I finally got what you mean. Yeah, this is good. Please fix conflicts then I will merge it.

**Comment by staff0rd** [comment]
requested changes made

**Comment by bao-qian** [comment]
Hi, I merged this pr and fixed the conflict by git cherry-pick. ece497c :)

**Pull request closed without merging**